packages/reactive-element/src/decorators/query.ts

```diff
@@ -112,11 +114,24 @@ export function query(selector: string, cache?: boolean): QueryDecorator {
     } else {
       // This object works as the return type for both standard and
       // experimental decorators.
-      return {
+      resultDescriptor = {
+        ...descriptorDefaults,
         get(this: ReactiveElement) {
           return doQuery(this);
         },
       };
     }
+    if (typeof nameOrContext !== 'object' && extendedReflect.decorate) {
```

Comment:
Could a unit test be added? Although I assume this is tested internally via TAP so can punt.